com.serial_studio.Serial-Studio.yaml

```diff
@@ -0,0 +1,80 @@
+app-id: com.serial_studio.Serial-Studio
+runtime: org.kde.Platform
+runtime-version: '6.9'
+sdk: org.kde.Sdk
+command: serial-studio
+separate-locales: false
+rename-icon: serial-studio
+rename-desktop-file: serial-studio.desktop
+rename-appdata-file: serial-studio.metainfo.xml
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=all
+  - --allow=bluetooth
+  - --system-talk-name=org.bluez
```

Comment:
what is network, device=all and bluetooth for?

Reply:
@bbhtt Hi, I’m the author. The software interacts with a variety of input data sources, such as serial ports, network ports and Bluetooth LE to interface with embedded devices. It uses this data to build realtime dashboards. You can read more about it at https://serial-studio.com/